mpmath/libmp/libmpf.py

```diff
@@ -1472,6 +1492,11 @@ def format_fixed(s,
             s, max(precision+exponent+4, int(s[3]/blog2_10)), base)
     dps = precision + exponent + 1
 
+    if rounding == round_ceiling:
+        rounding = round_down if sign == '-' else round_up
+    elif rounding == round_floor:
+        rounding = round_up if sign == '-' else round_down
+
```

Comment:
This seems to be shared between two helpers.  Maybe this belongs to mpf_format()?